libs/wallet/src/connect-dialog/connect-dialog.tsx

```diff
@@ -28,15 +29,37 @@ type WalletType = 'gui' | 'cli' | 'hosted';
 
 export interface VegaConnectDialogProps {
   connectors: Connectors;
+  onChangeOpen?: (open: boolean) => void;
+}
+
+export const useVegaWalletDialogStore = create<VegaWalletDialogStore>(
```

Comment:
I think you completely didn't understand the purpose of this change. It **was** in trading app, (as well as separately in console-lite and token). And I moved it into lib. Now it's much more clearer and easier to use, because state of dialog comes outside app and don't interference wits global state of the app. Now this what you propose is just removing back this whole improvement.